api/api-doc/config.json

```diff
@@ -1,30 +1,59 @@
 "/v2/config/{id}": {
-      "get": {
```

Comment:
I guess its some automatic-formatter that processed this file behind the scenes. Please, drop this change, its unrelated here.